sdk/maps/maps-timezone-rest/package.json

```diff
@@ -13,97 +13,77 @@
+    "extract-api": "dev-tool run vendored rimraf review && dev-tool run vendored mkdirp ./review && dev-tool run extract-api",
+    "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
     "generate:client": "autorest --typescript swagger/README.md && npm run format",
-    "integration-test:browser": "dev-tool run test:browser",
-    "integration-test:node": "dev-tool run test:node-js-input -- --timeout 5000000 'dist-esm/test/**/*.spec.js'",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "lint:fix": "eslint package.json api-extractor.json src test --fix --fix-type [problem,suggestion]",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "dev-tool run test:vitest --esm",
     "lint": "eslint package.json api-extractor.json src test",
+    "lint:fix": "eslint package.json api-extractor.json src test --fix --fix-type [problem,suggestion]",
     "pack": "npm pack 2>&1",
+    "test": "npm run clean && npm run build:test && npm run unit-test",
     "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
     "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
-    "test": "npm run clean && npm run build:test && npm run unit-test",
     "unit-test": "npm run unit-test:node && npm run unit-test:browser",
-    "unit-test:node": "dev-tool run test:node-ts-input -- --timeout 1200000 --exclude 'test/**/browser/*.spec.ts' 'test/**/*.spec.ts'",
     "unit-test:browser": "echo skipped",
-    "build": "npm run clean && tsc -p . && dev-tool run bundle && mkdirp ./review && api-extractor run --local"
+    "unit-test:node": "dev-tool run test:vitest"
   },
   "sideEffects": false,
   "autoPublish": false,
   "dependencies": {
-    "@azure-rest/core-client": "2.3.1",
-    "@azure/core-auth": "^1.3.0",
-    "@azure/core-rest-pipeline": "^1.8.0",
-    "@azure/core-util": "~1.9.2",
-    "@azure/logger": "^1.0.0",
-    "@azure/maps-common": "~1.0.0-beta.3",
-    "tslib": "^2.2.0"
+    "@azure-rest/core-client": "^2.3.1",
+    "@azure/core-auth": "^1.9.0",
+    "@azure/core-lro": "^2.7.2",
+    "@azure/core-rest-pipeline": "^1.18.0",
+    "@azure/logger": "^1.1.4",
+    "@azure/maps-common": "1.0.0-beta.2",
+    "@azure/core-util": "^1.11.0",
+    "tslib": "^2.8.1"
   },
   "devDependencies": {
-    "@azure-tools/test-credential": "^1.0.0",
-    "@azure-tools/test-recorder": "^3.0.0",
+    "@azure-tools/test-credential": "^2.0.0",
+    "@azure-tools/test-recorder": "^4.1.0",
+    "@azure-tools/test-utils-vitest": "^1.0.0",
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure/identity": "^2.0.1",
     "@microsoft/api-extractor": "^7.31.1",
```

Comment:
remove "@microsoft/api-extractor": "^7.31.1", and replace `api-extractor run --local` with `dev-tool run extract-api`